toastr.js

```diff
@@ -346,13 +349,18 @@
                 }
 
                 function hideToast(override) {
+                    var opts = {};
+                    ['Method', 'Duration', 'Easing'].forEach(function(opt) {
```

Comment:
Absolutely. Just so I'm clear, you mean rather than the loop, doing something akin to this?:

```js
var closeMethod = override && options.closeMethod !== false ? options.closeMethod : options.hideMethod;
var closeDuration = override && options.closeDuration !== false ? options.closeDuration : options.hideDuration;
var closeEasing = override && options.closeEasing !== false ? options.closeEasing : options.hideEasing;
```